plugins/vpp/ifplugin/vppcalls/vpp2001/dump_interface_vppcalls.go

```diff
@@ -1151,7 +1151,9 @@ func ifAPIFlagState(flags vpp_ifs.IfStatusFlags) uint32 {
 	if flags == vpp_ifs.IF_STATUS_API_FLAG_ADMIN_UP {
 		return 1
 	} else if flags == vpp_ifs.IF_STATUS_API_FLAG_LINK_UP {
-		return 2
+		return 0
+	} else if flags == 3 {
+		return 3
```

Comment:
It is probably a VPP bug that the "value 3" is not defined in the binary APIs. If it has not been changed by mistake (try to find the commit that introduced the change), please create a Jira issue on fd.io wiki. 
Also not sure if returning 0 for IF_STATUS_API_FLAG_LINK_UP is intended?

Reply:
Probably, because the "value 0" is not defined either, but it is returned in some cases. Actually, I've never seen `vpp_ifs.IF_STATUS_API_FLAG_ADMIN_UP` to be returned. For now it is always this three values
- 0
- IF_STATUS_API_FLAG_LINK_UP
- 3

Reply:
Isn't it determined by the bits?

- `0` => admin down, link down
- `1` => admin up, link down
- `2` => admin down, link up
- `3` => admin up, link up

Reply:
yes, bits is what matters. I've left a comment about it below